code/game/objects/structures/windoor_assembly.dm

```diff
@@ -46,7 +46,7 @@ obj/structure/windoor_assembly/Destroy()
 	icon_state = "[facing]_[secure]windoor_assembly[state]"
 
 /obj/structure/windoor_assembly/Cross(atom/movable/mover, turf/target, height=1.5, air_group = 0)
-	if(istype(mover) && mover.checkpass(PASSGLASS))
+	if(istype(mover) && (mover.checkpass(PASSDOOR) || mover.checkpass(PASSGLASS)))
```

Comment:
Because it's bitflags, `checkpass(A) || checkpass(B)` is equivalent to `checkpass(A|B)`